Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -673,7 +663,7 @@
 
         <controls:DataGrid
             x:Name="AllView"
-            Grid.Row="3"
+            Grid.Row="0"
```

Comment:
Wouldn't we remove this altogether if there are no rows/columns set on the Grid?